bin/d2l-test-runner.js

```diff
@@ -21,8 +22,15 @@ if (cli.subcommand === 'vdiff') {
 	} else if (vdiff.subcommand === 'migrate') {
 		const { migrate } = await import('../src/server/cli/vdiff/migrate.js');
 		await migrate.start(vdiff._unknown);
+	} else if (vdiff.subcommand === 'migrate-local') {
+
+		execSync('npm install @brightspace-ui/visual-diff@14  --no-save');
+		execSync('npx mocha \'./**/*.visual-diff.js\' -t 10000 --golden');
```

Comment:
Probably not a big deal in smaller repos, but in `core` this takes forever.  Would it work to accept a folder name (like `dialog`) and then do this:
```
execSync('npx mocha \'./**/dialog/*.visual-diff.js\' -t 10000 --golden');
```
and then in `migrate` do:
```
const dirs = await glob(`./**/${pattern}/*/${oldSuffix}`, { ignore: 'node_modules/**', posix: true });
```
?
Would solve the problem I have of having to run it ten times before I find the write pattern format